examples/meta/CMakeLists.txt

```diff
@@ -36,13 +36,19 @@ FOREACH(META_EXAMPLE ${META_EXAMPLES})
     SET(EXAMPLE_LISTINGS)
     SET(BASENAME ${EXAMPLE_REL_DIR}/${EXAMPLE_NAME})
     LIST(APPEND EXAMPLE_LISTINGS ${CMAKE_CURRENT_BINARY_DIR}/cpp/${BASENAME}.cpp)
+
+    # FIXME
+    # Hack to avoid generation of scala meta examples dependencies that will
+    # not be generated by generate.py script (since there is no scala.json)
+    LIST(REMOVE_ITEM AVAILABLE_INTERFACES "ScalaModular")
```

Comment:
Adding the Scala module to ```DISABLED_INTERFACES``` will cause to not include examples/meta/scala/CMakeLists.txt. Only the Scala integration tests will be generated (and they will fail) and the flag ```-DScalaModular``` will become a bit useless (it will behave like an alias of ```-DJavaModular=ON```). 

This hack permits to create all the Scala tests anyway (generated and integration), but as I said before, this is just a workaround to prevent the continuous generation of the meta examples code.